cpp/modules/luma.gl/garrow/src/array.cc

```diff
@@ -18,21 +18,25 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#include "./webgpu-column.h"  // NOLINT(build/include)
+#include "./array.h"  // NOLINT(build/include)
```

Comment:
In the JS API, the Column is a primary class that is visible to the app, `Table.getColumn` `Table.getColumnAt` etc. 

That is how we work with the tables.

Is that not the case in C++?

Reply:
Not sure I understand -- are you referring to the type name being used here, `Array`? There is no class named `Column` in C++.

Our `Table` API is identical to what `Arrow` provides, with the difference that `*column*` methods work with `ChunkedArray` instances, where as we skip the chunking step and work with `Array`s directly

Reply:
Understood, looks like the JS/C++ Arrow APIs are slightly more different than I expected. Which probably contributes to some of the confusion in our discussions :)